src/components/Dailog.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div class="main_container">
+    <div class="container" v-for="issue in issues" :key="issue.id">
+      <v-alert v-show="showAlert(issue)" class="dailog" type="error"
+        >{{ issue.bName }} to be returned on {{ issue.dueDate }}
+      </v-alert>
+    </div>
+  </div>
+</template>
+
+<script>
+import axios from "axios";
+export default {
+  name: "alertDailog",
+  data() {
+    return {
+      user: {},
+      issues: [],
+    };
+  },
+  methods: {
+    async loadData() {
+      let user = localStorage.getItem("user-info");
+      let res = await axios.get("http://localhost:3000/issues");
```

Comment:
can you load this data as follows:
-> create an action in vuex to get all the issues. 
-> store it in the store using appropriate functions.
-> use that data in this component

To use it in a component.
-> dispatch the getIssues action in one of the lifecycle hooks( read into which one you need to put it into)
-> Then use this issuedBooks data from the computed property.